node-perf-dash/kubelet-parser.go

```diff
@@ -188,7 +194,7 @@ func GrabTracingKubelet(d Downloader, job string, buildNumber int, nodeName stri
 			"test": sortedTestTime[currentTestIndex].TestName,
 			"node": nodeName,
 		},
-		Version: tracingVersion,
+		Version: perftype.CurrentKubeletPerfMetricsVersion,
```

Comment:
Maybe we could leave this empty now, and we don't need to expose `CurrentKubeletPerfMetricsVersion`, either.